types/foundry-vtt-api/types/apps/templates.d.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Get a template from the server by fetch request and caching the retrieved result
+ * @param path	The web-accessible HTML template URL
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/apps/templates.d.ts;
  Line: 3;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.